src/networking.c

```diff
@@ -2581,7 +2582,9 @@ void processInlineBuffer(client *c) {
 
     /* Create an Object for all arguments. */
     for (c->argc = 0, j = 0; j < argc; j++) {
-        c->argv[c->argc] = createObject(OBJ_STRING, argv[j]);
+        robj *obj = createObject(OBJ_STRING, argv[j]);
+        obj->allocator_id = getCurThreadID();
+        c->argv[c->argc] = obj;
```

Comment:
I wonder if it wouldn't be better to set the allocatorId for every created object? it would remove some code duplications